portfolio/src/main/webapp/script.js

```diff
@@ -97,3 +97,11 @@ function createElement(text, type) {
   element.innerText = text;
   return element;
 }
+
+function validateForm() {
+  var comment = document.forms["comment-section"]["comment-input"].value;
+  if (comment == "") {
+    alert("Comment cannot be empty :)");
+    return false;
```

Comment:
Will returning `false` here prevent posting the comment to the server? Can you please try it and let me know what will be the result? :)

Reply:
yes! tried it, here is the result if i submit emptycomment, nothing is shown in the comments section
![image](https://user-images.githubusercontent.com/41379082/88318481-afc12e80-cd12-11ea-9326-3275f1a595f0.png)

Reply:
Cool! Thanks for confirming 👍